content/en/docs/concepts/signals/traces.md

```diff
@@ -12,6 +12,10 @@ sophisticated mesh of services, traces are essential to understanding the full
 
 Let's explore this with three units of work, represented as [Spans](#spans):
 
+> 📝 **Note**: The JSON examples below do not represent a particular format, 
+> and especially not OTLP JSON which is much more verbose.
+
+
```

Comment:
```suggestion
{{% alert title="Note" %}}

The JSON examples below do not represent a particular format, 
and especially not OTLP JSON which is much more verbose.

{{% /alert %}}
```

Although the `>` renders as a note as well using the alert shortcode is the preferred way, because it gives us some flexibililty in color, title, etc.

Reply:
Will do. What is the preferred link for adding here, please?